utils/format-date.js

```diff
@@ -0,0 +1,22 @@
+export const formatDate = (input) => {
+    const date = new Date(Date.parse(input));
+    let mon = new Array();
+    mon[0] = "Jan";
+    mon[1] = "Feb";
+    mon[2] = "Mar";
+    mon[3] = "Apr";
+    mon[4] = "May";
+    mon[5] = "Jun";
+    mon[6] = "Jul";
+    mon[7] = "Aug";
+    mon[8] = "Sep";
+    mon[9] = "Oct";
+    mon[10] = "Nov";
+    mon[11] = "Dec";
+    let month = mon[date.getMonth()];
+    return {
+    day: date.getDate(),
+    month: month,
```

Comment:
Instead of creating a `mon` array. You can do `date.toLocaleString('default', { month: 'long' });`